src/status_im/ui/screens/communities/styles.cljs

```diff
@@ -29,40 +29,31 @@
 
 (defn card-stats-container []
   {:flex-direction :row
```

Comment:
general approach here was to make styling relative here and add some parameters to the functions so they can be reused